addons/l10n_it_edi_sdicoop/models/account_edi_format.py

```diff
@@ -209,13 +216,15 @@ def _l10n_it_post_invoices_step_2(self, invoices):
                 to_return[invoice] = {'error': _('You are not allowed to check the status of this invoice.'), 'blocking_level': 'error'}
                 continue
 
-            xml = proxy_user._decrypt_data(response['file'], response['key'])
-            response_tree = etree.fromstring(xml)
+            if response.get('file'):
+                xml = proxy_user._decrypt_data(response['file'], response['key'])
+                response_tree = etree.fromstring(xml)
             if state == 'ricevutaConsegna':
                 to_return[invoice] = {'error': _('The invoice has been succesfully transmitted. The addressee has 15 days to accept or reject it.')}
-            elif state == 'notificaScarto':
+            elif state == 'notificaScarto' and response.get('file'):
                 errors = [element.find('Descrizione').text for element in response_tree.xpath('//Errore')]
-                to_return[invoice] = {'error': self._format_error_message(_('The invoice has been refused by the Exchange System'), errors), 'blocking_level': 'error'}
+                to_return[invoice] = {'error': self._format_error_message(_('The invoice has been refused by the Exchange System'), errors),
+                                      'blocking_level': 'error'}
```

Comment:
Is there a real diff? If it is just for formatting, why not, but the line is still pretty long...